scripts/dts/devicetree.py

```diff
@@ -48,15 +48,23 @@ def parse_node_name(line):
         addr = None
 
     if ':' in line:
-        label, name = line.split(':')
+        if len(line.split(':')) == 3:
+            atl_label, label, name = line.split(':')
```

Comment:
Should be `alt_label` instead of `atl_label`.

Reply:
Indeed, thanks